internal/integ/client/client.go

```diff
@@ -0,0 +1,18 @@
+package client
```

Comment:
IMHO, I don't know if it's really necessary, to create a client package for a small amount of logic, that could stay as a helper func?